apps/nextra/pages/en/build/cli/trying-things-on-chain/ledger.mdx

```diff
@@ -117,3 +119,495 @@ yes
 After you have approved publishing this package you will be prompted to sign the transaction on your Ledger device. Once signed, the package will be published to the network!
 
 One error you might run into is `Error: Wrong raw transaction length`. This means that the transaction or package size was too big for your device to sign. Currently the Aptos Ledger app can only support transactions that are smaller than 20kb. The `Ledger Nano S` device has less memory than that, which is why it is more likely to produce this error.
+
+## Authentication key rotation
+
+If you have an active account that is not secured using a hardware wallet, then
+you may wish to rotate the account's authentication key so that it corresponds
+to a [BIP44 account index] private key held on your Ledger.
+
+Alternatively, if you have an account linked with a Ledger hardware wallet that
+you wish to publish a large package from, you might want to temporarily rotate
+the account's authentication key to a hot key to avoid memory issues.
+
+This tutorial will walk you through both scenarios.
+
+<Callout type="warning" emoji="💀">
+Before you start this tutorial make sure you have completed the
+[key rotation guide](../../advanced-guides/key-rotation.mdx).
+</Callout>
+
+<Steps>
+
+### Complete the key rotation guide
+
+Confirm that you have completed the
+[key rotation guide](../../advanced-guides/key-rotation.mdx).
+
+### Verify your Ledger is ready
+
+1. Connect and unlock your Ledger.
+1. Check what version of the Aptos app you have: `Aptos > About > Version`.
+1. If you do not have version `0.6.9` or higher, update it using Ledger Live.
+1. Enable blind signing: `Aptos > Settings > Enable Blind Signing`.
+
+### Start a localnet
+
+Start a localnet:
+
+```sh filename="Terminal"
+aptos node run-localnet
+```
+
+The localnet is ready when it prints out:
+
+```sh filename="Terminal"
+Applying post startup steps...
+
+Setup is complete, you can now use the localnet!
+```
+
+<Callout type="info" emoji="🧠">
+If you are on a UNIX-like system, the following command can be used to start a
+fresh localnet as a background process:
+
+```sh filename="Terminal"
+mkdir -p localnet-data
+aptos node run-localnet \
+    --assume-yes \
+    --test-dir localnet-data \
+    --force-restart &
+export LOCALNET_PID=$!
+```
+
+You can then stop the localnet at any point with the following command:
+
+```sh filename="Terminal"
+kill $LOCALNET_PID
+```
+</Callout>
+
+### Set up localnet hot wallet profile
+
+Create a private key corresponding to an authentication key, and thus initial
+account address, that starts with the vanity prefix `0xaaa`:
+
+```sh filename="Terminal"
+aptos key generate \
+    --assume-yes \
+    --output-file private-key-a \
+    --vanity-prefix 0xaaa
+```
```

Comment:
It can actually be kinda hard to follow along if luck of the draw two accounts start with similar hex chars

Vanity address with only 3 chars is negligible time to generate, and the labelling make the tutorial flow way easier